src/huntsman/pocs/utils/huntsman.py

```diff
@@ -16,6 +16,7 @@
 from huntsman.pocs.dome import create_dome_from_config
 from huntsman.pocs.core import HuntsmanPOCS
 from huntsman.pocs.scheduler.constraint import SunAvoidance
+from huntsman.pocs.scheduler.constraint import AlreadyVisited
```

Comment:
This is in the PANOPTES POCS. This should fix the failing test (hopefully)

```suggestion
from panoptes.pocs.scheduler.constraint import AlreadyVisited
```